cocos/2d/assets/sprite-frame.ts

```diff
@@ -1438,9 +1439,24 @@ export class SpriteFrame extends Asset {
         sp._texture = this._texture;
         sp._isFlipUVX = this._isFlipUVX;
         sp._isFlipUVY = this._isFlipUVY;
+        if (this._original) {
+            sp._original = {
+                _texture: this._original._texture,
+                _x: this._original._x,
+                _y: this._original._y,
+            };
+        } else {
+            sp._original = null;
+        }
+        sp._packable = this._packable;
         sp._pixelsToUnit = this._pixelsToUnit;
         sp._pivot.set(this._pivot);
         sp._meshType = this._meshType;
+        sp._extrude = this._extrude;
+        sp._customOutLine.splice(0, sp._customOutLine.length, ...this._customOutLine);
+        sp._mesh = this._mesh;
```

Comment:
I'm not sure it's needed here. mesh currently doesn't implement the clone function.

Reply:
It is not a clone because they share the same mesh. What i think clone is a deep copy.